app/controllers/payroll_controller.py

```diff
@@ -0,0 +1,12 @@
+from app.database.models import Payroll
+
+class PayrollController:
+    def weeklyPaid(self, hours, rate):
+	if hours > 40:
+		return 40 * rate + (hours - 40) * rate * 1.5
+	else:
+		return hours * rate
```

Comment:
Is this temporary? If so, please remove this file.

Reply:
No just practice. I will remove later.